.gitignore

```diff
@@ -20,9 +20,9 @@ node_modules
 /fonts
 /js
 /contents/testing
-sgds/sgds.css.map
+css/sgds.css.map
 assets/js/*.js
 assets/js/*.js.map
 assets/sgds.js
 assets/sgds.js.map
-sgds/sgds.css
+css/sgds.css
```

Comment:
To untrack ./css folder correctly, 

1).gitignore the entire /css folder instead 

2) and on your local terminal run  `git rm -r --cached ./css`

3) commit and push your changes 

- this will untrack the entire css folder and not get committed (since it doesnt need to be committed). Currently css folder is still being tracked by git